docs/docs/plugins/README.md

```diff
@@ -1,16 +1,14 @@
 # Plugins
 
-- [Getting Started](/plugins/getting-started.md)
-- [Plugins List](/plugins/list.md)
-- [Plugins Technical Details](/plugins/details.md)
+- [Getting Started](getting-started.md)
+- [Plugins List](list.md)
+- [Plugins Technical Details](details.md)
```

Comment:
I don't think we want to make these changes. The formatting/links don't work in GitHub, but it's not meant to, this `docs/` directory is meant to be viewed on the [docs site](https://projects.laion.ai/Open-Assistant/docs/plugins/).

Reply:
Ohhhh my bad. Oops. Canceling.